src/lib/jsonParse/jsonUpdateContextRequest.cpp

```diff
@@ -389,11 +389,13 @@ std::string jsonUpcrCheck(ParseData* reqData, ConnectionInfo* ciP)
 */
 void jsonUpcrPresent(ParseData* reqDataP)
 {
+  // FIXME PR: leftover?
   if (!lmTraceIsSet(LmtPresent))
```

Comment:
@kzangeli is this a leftover?

Reply:
Seems like it.
However ... looking a little closer, the function is called jsonUpcr**Present**.
It does nothing but presenting info. And always using the Log trace `LmtPresent`, do, if that log level is not set, why spend any time inside this function?

So no, not a leftover.

Reply:
Thus, the following removed lines should be recovered also, right? https://github.com/telefonicaid/fiware-orion/pull/3197/files#diff-2635eb6d54466f77bca7735f68e47ee3L130

A bit off-topic but... how are we useing *present() methods at the present moment? I think we were using in the very early days of CB to debug, but I'm not pretty sure right now... In which points of the code are we invoking "root" *present() methods?

Reply:
Not sure you read more than the first line of my comment ...
It is **not** a leftover.
But then again, as you say, the `present`  methods are no longer used (at least not by me, and I'm the one who invented them ... :))
I see no problem with removing all of them.

Reply:
> Not sure you read more than the first line of my comment ...
> It is not a leftover.

Yes, I dit read... that's why I'm saying that if it is **not** a leftover then the lines I removed at https://github.com/telefonicaid/fiware-orion/pull/3197/files#diff-2635eb6d54466f77bca7735f68e47ee3L130 thinking that part was also a leftover should be restored. 

I haven't done yet, but I will. I just what waiting to conclude the discussion on this comment.

> But then again, as you say, the present methods are no longer used (at least not by me, and I'm the one who invented them ... :))
> I see no problem with removing all of them.

I though that maybe at some point in parse library we have some invocation to present() under some conditions (e.g. `LmtPresent` activated). But if they are no longer called from any place and they are dead code, yest, better to remove.

However, not in this PR. This PR is about camelCase for actionTypes, so it is out my intention to touch too much other not related parts (as present() logic is).

Reply:
Yes, not in thiCasa de aperosCasa de apeross PR.
Remove the "leftover" comment. It is incorrect.
And let's create an issue about removing all `present` methods

Reply:
Fixed in 2a3c07a3159e28aefe070e317edd6d10ac219f62. And issue created at https://github.com/telefonicaid/fiware-orion/issues/3199